plugin/src/main/java/io/jenkins/plugins/analysis/warnings/CodeChecker.java

```diff
@@ -0,0 +1,33 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+
+/**
+ * Provides a parser and customized messages for the clang-analyzer.
```

Comment:
Does this refer to clang-analyzer by mistake?

Reply:
Yes, you are right.
I am going to fix that